src/DegenGambit.sol

```diff
@@ -795,4 +795,15 @@ contract DegenGambit is ERC20, ReentrancyGuard {
 
         prize = payout(left, center, right);
     }
+
+    /// isDebuggable pure function that returns a string with debug version
+    /// if empty string non-debuggable
+    function debuggerVersion()
```

Comment:
I don't like that the main contract has something called `debuggerVersion`.

Why not have a simple `version()` method?

For production contracts, we could just have versions like `"1"`, `"42"`, etc. (I don't think we need semantic versioning here.)

For debuggable contracts, we can use `"1-debug"`, `"42-debug"`, etc.

Clients can parse these to decide on behavior.

Reply:
Took the suggestion made it version instead. Looks cleaner and nicer